lib/readify.js

```diff
@@ -31,24 +31,77 @@ const nicki = !WIN && !BROWSER && require('nicki/legacy');
 const readdir = promisify(fs.readdir, fs);
 
 /* sorting on Win and node v0.8.0 */
-const sortFiles = sort((a, b) => {
-    return a.name > b.name ? 1 : -1;
+const sortFiles = currify((attr, order, array) => {
+    const cmpCallbacks = {
+        'numeric': (a, b) => (+a - +b),
+        'local_string': (a, b) => a.localeCompare(b.attr),
+        'default': (a, b) => (a > b ? 1 : -1),
```

Comment:
What difference between `default` and `numeric` sorting?
What purpose of `localeCompare`? As I understand from [mdn](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) without `locales` argument it will work in the same way as `default` sorting.